apps/admin/backend/src/tabulation/full_results.ts

```diff
@@ -82,6 +82,7 @@ const tabulateCastVoteRecordsMemoized = memoize(
  * Clears the memoized results of tabulating cast vote records.
  */
 export function clearTabulationCache(): void {
+  /* c8 ignore next 1 */
```

Comment:
I don't know why exactly `clear` is typed as an optional property of the memoized function's cache. I've tested and found it always exists, even immediately after the function is initialized.

Because `clear` doesn't work outside of its `this` context, we can't `assertDefined(...)()`. We could typecast, use an eslint ignored non-null assertion, or skip coverage, and this seemed the cleanest and safest.

Reply:
It's because you can change the caching mechanism to something that doesn't have `clear`, e.g. `WeakMap` (see [this commit](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/6448efea6b05d28d3f77f36c7c9e161af18788ea)).